server/build/plugins/json-pages-plugin.js

```diff
@@ -0,0 +1,22 @@
+export default class JsonPagesPlugin {
```

Comment:
@nkzawa This is my first webpack plugin.
Could you check whether I have done something ugly here?